docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -7,6 +7,107 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+[float]
+=== 7.12.0
+
+<<access-to-keychain-credentials-directories>>
+
+<<attempt-to-remove-file-quarantine-attribute>>
+
+<<azure-automation-account-created>>
+
+<<azure-automation-runbook-created-or-modified>>
+
+<<azure-automation-runbook-deleted>>
+
+<<azure-automation-webhook-created>>
+
+<<azure-blob-container-access-level-modification>>
+
+<<azure-command-execution-on-virtual-machine>>
+
+<<azure-diagnostic-settings-deletion>>
+
+<<azure-event-hub-authorization-rule-created-or-updated>>
+
+<<azure-event-hub-deletion>>
+
+<<azure-firewall-policy-deletion>>
+
+<<azure-key-vault-modified>>
+
+<<azure-network-watcher-deletion>>
+
+<<azure-resource-group-deletion>>
+
+<<azure-storage-account-key-regenerated>>
+
+<<connection-to-commonly-abused-web-services>>
+
+<<credential-acquisition-via-registry-hive-dumping>>
+
+<<execution-from-unusual-directory-command-line>>
+
+<<execution-with-explicit-credentials-via-scripting>>
+
+<<file-and-directory-discovery>>
+
+<<installation-of-custom-shim-databases>>
+
+<<outbound-scheduled-task-activity-via-powershell>>
+
+<<persistence-via-microsoft-office-addins>>
+
+<<persistence-via-microsoft-outlook-vba>>
+
+<<persistence-via-update-orchestrator-service-hijack>>
+
+<<persistence-via-wmi-event-subscription>>
+
+<<potential-command-and-control-via-internet-explorer>>
+
+<<potential-remote-desktop-tunneling-detected>>
+
+<<potential-secure-file-deletion-via-sdelete-utility>>
+
+<<prompt-for-credentials-with-osascript>>
+
+<<remote-ssh-login-enabled-via-systemsetup-command>>
+
+<<scheduled-task-created-by-a-windows-script>>
+
+<<service-command-lateral-movement>>
+
+<<setuid-setgid-bit-set-via-chmod>>
```

Comment:
how do we deal with deprecated rules? this one is deprecated but listed as an updated query. i don't think we yet have deprecation built into these docs.

Reply:
> Couldn't leave an in-line comment because wasn't sure where to enter it so I took a screenshot instead. I think the word "launched" might be misspelled in the title and description of the "Suspicious Hidden Child Process of Launchd" rule--both are highlighted to in the following image. Everything else looks good!
> 
> <img alt="mispelling" width="816" src="https://user-images.githubusercontent.com/79124755/112059698-8b805080-8b32-11eb-9d49-d1a360150a8d.png">

Thanks for finding this @nastasha-solomon. In this case, it is referring to a specific process names `launchd` so it is correct

Reply:
Ah ok. Thanks for clarifying!